pass-journal-loader-integration/src/test/java/org/dataconservancy/pass/loader/journal/integration/DepositIT.java

```diff
@@ -96,14 +96,16 @@ public void loadFromFileTest() throws Exception {
 
         load = jar(new File(System.getProperty("nih.loader.jar").toString()))
                 .logOutput(LoggerFactory.getLogger("nih-loader"))
-                .withEnv("pmc", DepositIT.class.getResource("/pmc-2.csv").getPath())
-                .withEnv("pass.fedora.baseurl", PASS_BASEURL)
-                .withEnv("LOG.org.dataconservancy.pass", "DEBUG")
+                .withEnv("PMC", DepositIT.class.getResource("/pmc-2.csv").getPath())
+                .withEnv("PASS_FEDORA_BASEURL", PASS_BASEURL)
+                .withEnv("LOG_ORG_DATACONSERVANCY_PASS", "DEBUG")
                 .start();
 
+        wait(load);
+
         // The last dataset removed a type A journal, so now we expect only one
         assertEquals(3, listJournals().size());
-        assertEquals(2, typeA(listJournals()).size());
+        assertEquals(1, typeA(listJournals()).size());
```

Comment:
What's the deal here - I don't see a change to the test data that suggests the list of "type A" journals changes.  Then again, I don't fully remember what the CSV has in it.  Can you confirm that the type A stats are correct?